tests/components/airzone/test_climate.py

```diff
@@ -551,6 +552,7 @@ async def test_airzone_climate_set_temp(hass: HomeAssistant) -> None:
             SERVICE_SET_TEMPERATURE,
             {
                 ATTR_ENTITY_ID: "climate.dorm_2",
+                ATTR_HVAC_MODE: HVACMode.HEAT,
```

Comment:
We should assert something actually changed by this

Reply:
> We should assert something actually changed by this

Done in a419cd4a047a97f85e6b5015f855276a75da51d6